modules/virt-importing-rhel-image-boot-source-web.adoc

```diff
@@ -27,4 +27,5 @@ You can import a Red Hat Enterprise Linux (RHEL) image as a boot source by speci
 . In the *Add boot source to template* window, paste the copied URL of the guest image into the *Import URL* field, and click *Save and import*.
 
 .Verification
-* In the *Templates* tab, verify that the *Boot source* column displays *Available* for the RHEL virtual machine template. You can use this template to create RHEL virtual machines.
+
+* In the *Templates* tab, verify that the tile for this template displays a green checkmark to confirm that this template already has a boot source added. You can use this template to create RHEL virtual machines.
```

Comment:
To make this easier to read, I'd suggest breaking up this step into shorter bits. Suggested rewrite:

```
To verify that a boot source was added to the template:

. Click the *Templates* tab.

. Confirm that the tile for this template displays a green checkmark. 

You can now use this template to create RHEL virtual machines.
```

What do you think?

Reply:
Makes it cleaner! Removes the clumping! Updated the content.